packages/flutter/lib/firebase.dart

```diff
@@ -0,0 +1,11 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Service exposed to Flutter apps that implements a subset of the Firebase
+/// API.
+///
+/// This library will probably be moved into a separate package eventually.
+library firebase;
```

Comment:
Yeah. I'm not even sure any of this should be in the flutter/flutter/ or flutter/engine/ repos. I think this, and other services like GCM, should be in a separate repo, say flutter/third-party-sdks/.

cc @mpcomplete since he might have opinions also given what he's working on.